reference/api/health_api.md

```diff
@@ -0,0 +1,33 @@
+# Dapr health API reference
+
+Dapr provides health checking probes that can be used as readiness or liveness of Dapr.
+
+### Get Dapr health state
+
+Gets the health state for Dapr.
+
+#### HTTP Request
+
+```http
+GET http://localhost:<daprPort>/v1.0/healthz
+```
+
+#### HTTP Response Codes
+
+Code | Description
+---- | -----------
+200  | health is ok
```

Comment:
I would change to `dapr is healthy` and `dapr is not healthy`

Reply:
Other than that, LGTM

Reply:
Done as suggested. thanks.